product-accelerators/apim/components/org.wso2.healthcare.apim.backendauth/src/main/java/org/wso2/healthcare/apim/backendauth/BackendAuthenticator.java

```diff
@@ -104,12 +118,48 @@ public boolean mediate(MessageContext messageContext) {
         return true;
     }
 
-    public String getConfigName() {
-        return configName;
+    @SuppressWarnings("Setters will be called by the ESB config builder")
```

Comment:
is this needed?

Reply:
removed in [b459794](https://github.com/wso2/healthcare-accelerator/pull/15/commits/b4597940253b7b805fa7e95d2c2cdb0e7216d947)